prolog/uglify_ease.pl

```diff
@@ -14,14 +14,22 @@
 
 ease_assert(Graph, rdf(_S,P,O)) :-
   rdf_equal(P,owl:imports),!,
-  ( rdf_split_url('http://www.ease-crc.org/ont/',_,O) -> true ; (
-    ease_ugly_ontology(EASE_UGLY),
-    rdf_assert(EASE_UGLY,P,O,Graph)
-  )).
+  ease_assert_import(Graph,O).
 
 ease_assert(Graph, rdf(S,P,O)) :-
   rdf_assert(S,P,O,Graph).
 
+%%
+ease_assert_import(Graph,Ontology) :-
```

Comment:
`Graph` is now a Singleton variable and could thus be `_Graph` to denote it's essentially not used.